### PR TITLE
added Data model v2

### PR DESCRIPTION
**Pull request by amoujaes1**

### CODE REVIEW

File: clearblade.md [hunk 1]
```diff
@@ -1186,6 +1187,42 @@ Retrieves the count of items in a collection that match the supplied query
...
+		var db = ClearBlade.Database();
+~~~
+
+## ClearBlade.Database.query(_query, callback)
+
+This function returns query results to be parsed.
+
+~~~~javascript
+	var db = ClearBlade.Database();
+	db.query("select sum(ingress) from traffic where entrance='Main_Entrance';", callback);
```
Comment (rreinold) [inline clearblade.md]: callback must be defined before referencing it here

File: clearblade.md [hunk 2]
```diff
@@ -1186,6 +1187,42 @@ Retrieves the count of items in a collection that match the supplied query
...
+    };
+~~~~
+
+## ClearBlade.Database.exec(_query, callback)
+
+This function does not return query results. 
+
+~~~~javascript
+	var db = ClearBlade.Database();
+	db.exec("delete from traffic where entrance='Store_Entrance';");
```
Comment (rreinold) [inline clearblade.md]: Is there a callback here?

Comment (rreinold) [reply clearblade.md]: confirmed, yes there should be a callback

File: clearblade.md [hunk 3]
```diff
@@ -1186,6 +1187,48 @@ Retrieves the count of items in a collection that match the supplied query
 
 ~~~
 
+# Database
+
+Class: ClearBlade.Database();
```
Comment (yashjain28) [inline clearblade.md]: Please maintain this format:
https://github.com/ClearBlade/native-libraries/blob/master/clearblade.md#messagingsubscribetopic-callback

implies: add jsdoc for the query method, callback & also show the structure of the returned data.

File: clearblade.md [hunk 4]
```diff
@@ -1186,6 +1187,48 @@ Retrieves the count of items in a collection that match the supplied query
...
+        	resp.error("Parse error : " + JSON.stringify(data));
+        } else {
+        	resp.success(data);
+        }
+	db.query("select sum(ingress) from traffic where entrance='Main_Entrance';", callback);
+	//'Ingress' is a column in 'traffic' collection
+    };
+~~~~
+
+## ClearBlade.Database.exec(_query, callback)
```
Comment (yashjain28) [inline clearblade.md]: If the release for the bug is not made until Friday, remove the exec. I'll merge it by Friday 26th Feb

**Pull request merged**